benchmarks/Fractions.Benchmarks/Fractions.Benchmarks.csproj

```diff
@@ -12,14 +12,6 @@
     <PackageReference Include="BenchmarkDotNet" Version="0.14.0" />
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-    </PackageReference>
-    <PackageReference Include="System.Reflection.Metadata" Version="9.0.0" />
-  </ItemGroup>
```

Comment:
I don't think this is currently necessary: I'm getting some binding redirect warnings for `System.Collections.Immutable` and the like, but nothing too concerning (hopefully we get an update from BDN with the new packages that came out last week).

Reply:
I fixed exactly these build warnings with it. See https://learn.microsoft.com/en-us/dotnet/framework/migration-guide/reference-assemblies

Reply:
I don't know, maybe it's because I already have the SDK installed and it's not giving me trouble- are you still seeing the warnings after swapping the targets?

Reply:
This is indeed unexpected behavior. But it's not really surprising when you consider the possibilities of Roslyn compiler extensions.